Modules/Devices/src/Devices.Infrastructure.Database.SqlServer/Migrations/20240408115400_SetDefaultSchema.cs

```diff
@@ -0,0 +1,191 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.Modules.Devices.Infrastructure.Database.SqlServer.Migrations
+{
+    /// <inheritdoc />
+    public partial class SetDefaultSchema : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.EnsureSchema(
+                name: "Devices");
+
+            migrationBuilder.RenameTable(
+                name: "Tiers",
+                newName: "Tiers",
+                newSchema: "Devices");
```

Comment:
What's happening here? Do these things make sense?
It seems EF Core understands that the schema has changed, but it has not, actually.

Reply:
See PR description.

Reply:
I did. 

> The generated migrations' "Up" methods don't really do anything

Why are they needed then?

Reply:
Well, imagine in the future there is a new database, and there the users don't have a default schema configured. When we now execute all migrations, the first ones will create the tables in the `dbo`/`postgres` schema, because that's the default default schema - if you know what I mean.

Though as you can see I now deleted the migration in the Admin API project. So there will be the same problem. But I didn't want to waste time figuring out what's necessary to move the views to a different schema (in postgres that is straightforward, but in SQL Server it's not). And since we will soon combine all the migrations anyway, I decided to ignore this potential problem. Currently there is no short-term plan to have a new database anyway.

Reply:
So you're saying that the code I highlighted takes care of moving the table to the correct schema? Will that work well with references to other tables?

Why don't we simply change the migrations where tables are created and specify the correct schema there? I realise changing old migrations is not good practice, but I don't see the issue with it in this case.

Reply:
Tbh, I wanted to avoid the work necessary to do this. 
But I just tried to run the migrations in a new database, and they fail. So I will do exactly what you suggested.

Reply:
Done. You can have another look. I also wouldn't mind if you run the code on your machine to see if it works.